#! /bin/sh /usr/share/dpatch/dpatch-run
## 01-configure.dpatch by Stuart Brady <sdbrady@ntlworld.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Rename to fuse-emulator, to avoid conflicting with
## DP: the Filesystem in USErspace packages.

@DPATCH@
--- fuse-emulator-0.9.0-old/configure.in	2008-01-12 21:02:32.000000000 +0000
+++ fuse-emulator-0.9.0-new/configure.in	2008-01-08 22:15:59.000000000 +0000
@@ -24,7 +24,7 @@
 AC_CANONICAL_SYSTEM
 
 dnl Use automake to produce `Makefile.in'
-AM_INIT_AUTOMAKE(fuse, 0.9.0)
+AM_INIT_AUTOMAKE(fuse-emulator, 0.9.0)
 
 dnl Checks for programs.
 AC_PROG_CC
   fi
 fi
 
